src/logger.js

```diff
@@ -119,19 +112,26 @@ function addLogger(parent, name, createFN, cfg, params) {
  *                                            `rootLogger`.
  */
 function setupHelixLogger(params, logger = rootLogger) {
-  // only overwrite the default console logger if there is one
-  if (logger.loggers.has('default')) {
-    // replace default logger with our own console logger
-    logger.loggers.set('default', openhwiskConsoleLogger);
-  }
+  // add openwhisklogger to helix-log logger
+  if (!logger.loggers.has('openwhisk')) {
+    const owLogger = new OpenWhiskLogger({});
+    logger.loggers.set('openwhisk', owLogger);
```

Comment:
```suggestion
    logger.loggers.set('OpenWhiskLogger', owLogger);
```

Minor detail but other logger names are camel cased: `CoralogixLogger` and `PapertraiLogger`

Reply:
ok. updated.